Android/app/src/main/java/io/github/project_travel_mate/utilities/WorldClockActivity.java

```diff
@@ -40,6 +42,7 @@
     //private DigitalClock mDigitalClock;
 
     private static TextClock mDigitalClock;
+    private static final String mTag = "WorldClockActivity";
```

Comment:
remove changes from here

Reply:
I used this during debugging but forgot to remove after i finished. Now removed.